lib/patches/bundler/ruby_version.rb

```diff
@@ -0,0 +1,29 @@
+module Bundler
+  class RubyVersion
+    def self.system
+      ruby_engine         = if defined?(RUBY_ENGINE) && !RUBY_ENGINE.nil?
+                              RUBY_ENGINE.dup
+                            else
+                              # not defined in ruby 1.8.7
+                              "ruby"
+                            end
```

Comment:
This is very weirdly indented, I would rather keep the same indentation as the source to ease comparison.